codec-http/src/test/java/io/netty/handler/codec/http/websocketx/extensions/compression/PerMessageDeflateEncoderTest.java

```diff
@@ -53,7 +59,6 @@ public void testCompressedFrame() {
         // test
         assertNotNull(compressedFrame);
         assertNotNull(compressedFrame.content());
-        assertTrue(compressedFrame instanceof BinaryWebSocketFrame);
```

Comment:
Why was this removed ?

Reply:
https://github.com/netty/netty/blob/721d2c02da639bcc61cd1659921591018a4c7f3d/codec-http/src/test/java/io/netty/handler/codec/http/websocketx/extensions/compression/PerMessageDeflateEncoderTest.java#L57